app/src/org/commcare/util/externalizable/ImprovedPrototypeFactory.java

```diff
@@ -1,51 +1,26 @@
 package org.commcare.util.externalizable;
 
-import java.util.Hashtable;
-
 import org.javarosa.core.util.PrefixTree;

```

Comment:
Can you add @Override to everything in this class that is overriding child behavior